### PR TITLE
Multiple Audit Dates in Report

### PR DESCRIPTION
**Pull request by nllong**
See proposals/2019/Audit Date Enumerations.md

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -5724,10 +5724,34 @@
...
+                      <xs:documentation>Date of DateType enumeration. (CCYY-MM-DD)</xs:documentation>
+                    </xs:annotation>
+                  </xs:element>
+                  <xs:element name="DateType">
+                    <xs:annotation>
+                      <xs:documentation>Type of AuditDate.</xs:documentation>
+                    </xs:annotation>
+                    <xs:simpleType>
+                      <xs:restriction base="xs:string">
+                        <xs:enumeration value="Site Visit"/>
```
Comment (nllong) [inline BuildingSync.xsd]: Verify list of enums.

Comment (markborkum) [reply BuildingSync.xsd]: @nllong Can we add an "Other" or "Custom" value to the enumeration (similar to the definition of the `<auc:FloorAreas>` element)?

Comment (nllong) [reply BuildingSync.xsd]: @markborkum take a look now. I added custom, other enums, and a CustomDateType element

**Comment by jasondegraw** [comment]
Will merge this once CI comes back green.

**Pull request merged**